src/AppBundle/Form/Type/TaskType.php

```diff
@@ -52,7 +52,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('tags', 'tag', [
                 'required' => false,
-                'choices'  => $this->taskwarrior->tags()
+                'choices'  => $this->taskmanager->getTaskwarrior()->tags()
+            ])
+            ->add('dependencies', DependencyType::class, [
+                'required' => false,
+                'choices' => $this->taskmanager->createQueryBuilder()
```

Comment:
i think it has a big performance impact. i have over 300 tasks in my instance. if we hydrate all tasks it will be very slow.